### PR TITLE
Stripe feature

### PR DESCRIPTION
**Pull request by gxespino**
Fixed merge issues. Should work now. Don't forget to use figaro to populate heroku keys.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,6 @@
 
 # Ignore OSX system files.
 .DS_Store
+
+# Ignore application configuration
+/config/application.yml
```
Comment (emckenna) [inline .gitignore]: not sure this should be ignored

File: config/routes.rb [hunk 2]
```diff
@@ -31,7 +33,11 @@
   get "action" => "veterans#new"
 
   get "/launchparty" => redirect("http://www.eventbrite.com/e/operation-code-launch-party-tickets-16494832488")
+<<<<<<< HEAD
```
Comment (emckenna) [inline config/routes.rb]: looks like a unresolved merge made it into here.

File: test/models/donation_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class DonationTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (emckenna) [inline test/models/donation_test.rb]: :(

Comment (davidmolina) [inline test/models/donation_test.rb]: Ohhhh.

On Thu, Jul 2, 2015 at 1:07 PM, Eric Mckenna <notifications@github.com>
wrote:

> In test/models/donation_test.rb
> <https://github.com/OperationCode/operationcode/pull/73#discussion_r33817229>
> :
>
> > @@ -0,0 +1,7 @@
> > +require 'test_helper'
> > +
> > +class DonationTest < ActiveSupport::TestCase
> > +  # test "the truth" do
> > +  #   assert true
> > +  # end
> > +end
>
> :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OperationCode/operationcode/pull/73/files#r33817229>.
>



-- 
David
t. @davidcmolina <https://twitter.com/davidcmolina>
http://davidmolina.github.io/

File: db/schema.rb [hunk 4]
```diff
@@ -49,6 +49,15 @@
...
   add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
 
+  create_table "donations", force: :cascade do |t|
+    t.string   "first_name"
+    t.string   "email"
+    t.integer  "amount"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string   "last_name"
+  end
```
Comment (emckenna) [inline db/schema.rb]: probablay also need some sort of transaction id from the stripe response.

Comment (davidmolina) [inline db/schema.rb]: Trying it now.

On Thu, Jul 2, 2015 at 1:09 PM, Eric Mckenna <notifications@github.com>
wrote:

> In db/schema.rb
> <https://github.com/OperationCode/operationcode/pull/73#discussion_r33817388>
> :
>
> > @@ -49,6 +49,15 @@
> >    add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true, using: :btree
> >    add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
> >
> > +  create_table "donations", force: :cascade do |t|
> > +    t.string   "first_name"
> > +    t.string   "email"
> > +    t.integer  "amount"
> > +    t.datetime "created_at", null: false
> > +    t.datetime "updated_at", null: false
> > +    t.string   "last_name"
> > +  end
>
> probablay also need some sort of transaction id from the stripe response.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OperationCode/operationcode/pull/73/files#r33817388>.
>



-- 
David
t. @davidcmolina <https://twitter.com/davidcmolina>
http://davidmolina.github.io/

Comment (davidmolina) [inline db/schema.rb]: figaro to populate heroku keys?

On Thu, Jul 2, 2015 at 1:17 PM, David Molina <david.c.molina@gmail.com>
wrote:

> Trying it now.
>
> On Thu, Jul 2, 2015 at 1:09 PM, Eric Mckenna <notifications@github.com>
> wrote:
>
>> In db/schema.rb
>> <https://github.com/OperationCode/operationcode/pull/73#discussion_r33817388>
>> :
>>
>> > @@ -49,6 +49,15 @@
>> >    add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true, using: :btree
>> >    add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
>> >
>> > +  create_table "donations", force: :cascade do |t|
>> > +    t.string   "first_name"
>> > +    t.string   "email"
>> > +    t.integer  "amount"
>> > +    t.datetime "created_at", null: false
>> > +    t.datetime "updated_at", null: false
>> > +    t.string   "last_name"
>> > +  end
>>
>> probablay also need some sort of transaction id from the stripe response.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/OperationCode/operationcode/pull/73/files#r33817388>.
>>
>
>
>
> --
> David
> t. @davidcmolina <https://twitter.com/davidcmolina>
> http://davidmolina.github.io/
>



-- 
David
t. @davidcmolina <https://twitter.com/davidcmolina>
http://davidmolina.github.io/

Comment (davidmolina) [inline db/schema.rb]: Where do I get the pusher_app_id, pusher_key, and pusher_secret keys?

On Thu, Jul 2, 2015 at 1:20 PM, David Molina <david.c.molina@gmail.com>
wrote:

> figaro to populate heroku keys?
>
> On Thu, Jul 2, 2015 at 1:17 PM, David Molina <david.c.molina@gmail.com>
> wrote:
>
>> Trying it now.
>>
>> On Thu, Jul 2, 2015 at 1:09 PM, Eric Mckenna <notifications@github.com>
>> wrote:
>>
>>> In db/schema.rb
>>> <https://github.com/OperationCode/operationcode/pull/73#discussion_r33817388>
>>> :
>>>
>>> > @@ -49,6 +49,15 @@
>>> >    add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true, using: :btree
>>> >    add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
>>> >
>>> > +  create_table "donations", force: :cascade do |t|
>>> > +    t.string   "first_name"
>>> > +    t.string   "email"
>>> > +    t.integer  "amount"
>>> > +    t.datetime "created_at", null: false
>>> > +    t.datetime "updated_at", null: false
>>> > +    t.string   "last_name"
>>> > +  end
>>>
>>> probablay also need some sort of transaction id from the stripe response.
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/OperationCode/operationcode/pull/73/files#r33817388>
>>> .
>>>
>>
>>
>>
>> --
>> David
>> t. @davidcmolina <https://twitter.com/davidcmolina>
>> http://davidmolina.github.io/
>>
>
>
>
> --
> David
> t. @davidcmolina <https://twitter.com/davidcmolina>
> http://davidmolina.github.io/
>



-- 
David
t. @davidcmolina <https://twitter.com/davidcmolina>
http://davidmolina.github.io/

Comment (davidmolina) [inline db/schema.rb]: Disregard, found them. :)

On Thu, Jul 2, 2015 at 1:25 PM, David Molina <david.c.molina@gmail.com>
wrote:

> Where do I get the pusher_app_id, pusher_key, and pusher_secret keys?
>
> On Thu, Jul 2, 2015 at 1:20 PM, David Molina <david.c.molina@gmail.com>
> wrote:
>
>> figaro to populate heroku keys?
>>
>> On Thu, Jul 2, 2015 at 1:17 PM, David Molina <david.c.molina@gmail.com>
>> wrote:
>>
>>> Trying it now.
>>>
>>> On Thu, Jul 2, 2015 at 1:09 PM, Eric Mckenna <notifications@github.com>
>>> wrote:
>>>
>>>> In db/schema.rb
>>>> <https://github.com/OperationCode/operationcode/pull/73#discussion_r33817388>
>>>> :
>>>>
>>>> > @@ -49,6 +49,15 @@
>>>> >    add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true, using: :btree
>>>> >    add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
>>>> >
>>>> > +  create_table "donations", force: :cascade do |t|
>>>> > +    t.string   "first_name"
>>>> > +    t.string   "email"
>>>> > +    t.integer  "amount"
>>>> > +    t.datetime "created_at", null: false
>>>> > +    t.datetime "updated_at", null: false
>>>> > +    t.string   "last_name"
>>>> > +  end
>>>>
>>>> probablay also need some sort of transaction id from the stripe
>>>> response.
>>>>
>>>> —
>>>> Reply to this email directly or view it on GitHub
>>>> <https://github.com/OperationCode/operationcode/pull/73/files#r33817388>
>>>> .
>>>>
>>>
>>>
>>>
>>> --
>>> David
>>> t. @davidcmolina <https://twitter.com/davidcmolina>
>>> http://davidmolina.github.io/
>>>
>>
>>
>>
>> --
>> David
>> t. @davidcmolina <https://twitter.com/davidcmolina>
>> http://davidmolina.github.io/
>>
>
>
>
> --
> David
> t. @davidcmolina <https://twitter.com/davidcmolina>
> http://davidmolina.github.io/
>



-- 
David
t. @davidcmolina <https://twitter.com/davidcmolina>
http://davidmolina.github.io/

**Pull request merged**